docs/_includes/css/store.css

```diff
@@ -0,0 +1,243 @@
+.hero {
+  background: url({{site.url}}/assets/img/store-hero.png);
```

Comment:
add `{% assign path = site.url  | append: '/assets/img' %}` at the top of the file and use the path where necessary. e.g.
`background: url({{ path }}/store-hero.png);`